studio/schemas/cta.js

```diff
@@ -0,0 +1,73 @@
+export default {
+  title: 'Call to action',
```

Comment:
This is going to be super helpful to have and be able to drop on different pages